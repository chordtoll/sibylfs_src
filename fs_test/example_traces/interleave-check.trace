@type trace

Pid 2 -> create

open_close "foo" [O_CREAT] 0o600
Tau
RV_none

Pid 1 -> unlink "foo"
Pid 2 -> unlink "foo"
Tau
Tau
Pid 1 <- RV_none
Pid 2 <- ENOENT
