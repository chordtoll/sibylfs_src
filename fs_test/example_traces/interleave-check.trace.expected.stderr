
Error:     13: Pid 1 <- RV_none
   errors and values mixed in result
